web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-fifty-fifty.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ukxk2n.php, but found phpcs-scan-uKxK2n.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).